src/TodoList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+let todoList = [
+  { id: 1, title: "Homework" },
+  { id: 2, title: "Exercise" },
+  { id: 3, title: "Study" },
+];
+
+let list = todoList.map((item) => <li key={item.id}>{item.title}</li>);
+
```

Comment:
Not a bad idea but later on we'll probably use this expression right in the jsx component.

How about you try that in the next assignment 👍 
Nice implicit return here 🎖️